package.json

```diff
@@ -97,6 +97,7 @@
     "react-day-picker": "^8.7.1",
     "react-dom": "^18.2.0",
     "react-emoji-render": "^2.0.1",
+    "react-error-boundary": "^4.0.11",
```

Comment:
react-error-boundary will allow us to have a fallback view if we wrap a component with it that crashes.